i18n/en.json

```diff
@@ -851,6 +851,12 @@
   "admin.permissions.permission.manage_channel_roles.name": "Manage channel roles",
   "admin.permissions.permission.manage_emojis.description": "Create and delete custom emoji.",
   "admin.permissions.permission.manage_emojis.name": "Manage Custom Emoji",
+  "admin.permissions.permission.create_emojis.description": "Create custom emoji.",
+  "admin.permissions.permission.create_emojis.name": "Create Custom Emoji",
+  "admin.permissions.permission.delete_emojis.description": "Delete custom emoji.",
+  "admin.permissions.permission.delete_emojis.name": "Delete Custom Emoji",
+  "admin.permissions.permission.delete_others_emojis.description": "Delete others' custom emoji.",
+  "admin.permissions.permission.delete_others_emojis.name": "Delete Others' Custom Emoji",
```

Comment:
Do we need the descriptions if they simply echo the names or could they be left blank?

Reply:
I would say they must be different string, no matter if they are equal, because in the future they can change, and they can change independently. From my perspective they are different text that are temporary equal because are related, but aren't the same text.